src/main/webapp/firebase-chat.js

```diff
@@ -109,6 +109,12 @@ function authStateObserver(user) {
   }
 }
 
+function addSizeToGoogleProfilePic(url) {
```

Comment:
Optional: You could try parsing the url using https://developer.mozilla.org/en-US/docs/Web/API/URL/URL, which is cleaner. 

But, you would run into issues of not supporting Internet explorer.

Reply:
interesting, ill look into it